fixedwidth/src/main/java/org/apache/metamodel/fixedwidth/EbcdicReader.java

```diff
@@ -26,13 +26,17 @@
  */
 class EbcdicReader extends FixedWidthReader {
 
+    private final BufferedInputStream _stream;
+    private final String _charsetName;
     private final boolean _skipEbcdicHeader;
     private final boolean _eolPresent;
     private boolean _headerSkipped;
     
     public EbcdicReader(BufferedInputStream stream, String charsetName, int[] valueWidths,
             boolean failOnInconsistentLineWidth, boolean skipEbcdicHeader, boolean eolPresent) {
         super(stream, charsetName, valueWidths, failOnInconsistentLineWidth);
+        _stream = stream;
+        _charsetName = charsetName;
```

Comment:
Ah yes, it was fooling me because it used to be the super-class' fields and now they've moved here... Makes sense.